pages/home.tsx

```diff
@@ -1,9 +1,22 @@
-import React from "react";
-import type { NextApplicationPage } from "./_app";
+import React, { useState } from "react";
+import { CreateEventsWizard } from "../components/CreateEventsWizard";
+import styles from "../styles/Home.module.css";
+import { NextApplicationPage } from "./_app";
 //This is the page that is rendered when the 'Home' button from the Navbar is clicked
-
 const Home: NextApplicationPage = () => {
-  return <div>Home</div>;
+  const [showWizard, setShowWizard] = useState(false);
+  const handleClose = (): void => {
+    setShowWizard(false);
+  };
+  return (
+    <div className={styles.homeWrapper}>
+      <button className={styles.createBtn} onClick={() => setShowWizard(true)}>
+        Create
+        <img className={styles.addIcon} src="AddIcon.png" />
+      </button>
+      {showWizard ? <CreateEventsWizard handleClose={handleClose} /> : null}
+    </div>
+  );
```

Comment:
Like I mentioned in the comment, this should all go in the `AdminHomeView.tsx` component